api/Element.json

```
@@ -8147,7 +8148,8 @@
                     "name": "dom.w3c_pointer_events.enabled",
                     "value_to_set": "true"
                   }
-                ]
+                ],
+                "notes": "From Firefox 82 throws <code>NotFoundError</code> exception if pointer id is invalid (as per <a href='https://w3c.github.io/pointerevents/#setting-pointer-capture'>WC3 PointerEvents specification</a>). Earlier Firefox versions throw <code>InvalidPointerId</code>. See <a href='https://bugzil.la/1662124'>bug 1662124</a>."
```

We could do the same note here. Though I'm pretty suspicious of the existing data. Seems unlikely to me that the API is actually unsupported on Android for such a long time. Might be worth double checking whether we should mirror all the data from the desktop version.

```suggestion
                "notes": "Before Firefox 82, <code>setPointerCapture()</code> throws <code>InvalidPointerId</code> for an invalid <code>pointerId</code> argument. From Firefox 82, it throws <a href='https://w3c.github.io/pointerevents/#setting-pointer-capture'>the specified</a> <code>NotFoundError</code> exception. See <a href='https://bugzil.la/1662124'>bug 1662124</a>."
```

---

Good catch. I have raised the question on https://bugzilla.mozilla.org/show_bug.cgi?id=1662124#c9. Will commit both of these when that is confirmed.